docs/web-apps/automated-testing/cdp.md

```diff
@@ -108,11 +110,12 @@ options.platform_name = 'Windows 10';
 const sauce_options = {
   'username': SAUCE_USERNAME,
   'accessKey': SAUCE_ACCESS_KEY,
-  'devTools': true,
+  'devTools': true, // Legacy
```

Comment:
For the WDIO example, I'd use `webSocketUrl` specifically... I think it's worth mentioning that this option is enabled by default (please double-check this) in WDIO 9.